CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ### vNEXT
 
 - Allow `GraphQLRequestListener` callbacks in plugins to depend on `this`. [PR #2470](https://github.com/apollographql/apollo-server/pull/2470)
+- Default `privateVariables` and `privateHeaders` to 'true' to prevent sending potential PII to Apollo's cloud service
```

Comment:
FWIW I don't understand how versioning works between different packages in this repo. This maybe should be a major version bump if there wasn't other packages involved, but maybe it goes in lockstep with other packages? @abernix thoughts?

Reply:
🤷‍♂️

Reply:
Is changing default behaviour always considered a major version bump?

Reply:
@zionts 	@glasser 	This sure seems like it would be a major breaking change for Apollo Server as it changes the out-of-the-box behavior for everyone who hasn't configured this, no?  Apollo Server 3 is only _just_ on the (far) horizon, but would this be suitable to put there?

Reply:
Seems suitable to put it into AS3.0 @abernix , how do I do that?

Reply:
@zionts I've added it to the 3.x GitHub milestone so we keep tabs on it.

Reply:
While maybe this should be a major version bump for `apollo-engine-reporting` specifically, I don't think it needs to be tied up in waiting for the entire apollo-server project to get to a new major version. Yes, this is a backwards incompatible change, but I think there's something special about code that talks directly to our hosted services. We have an ongoing problem where people send us data by accident, and it's worth making the change faster rather than slower. If the protobufs were slightly different we could actually make this default change on our server side, and maybe we would have!

Note that my goal for this change is for the only place where it affects users (the trace inspector in Engine) to link to docs explaining how to enable sending trace values, and it's just a one line change (which can also be included in the CHANGELOG)... not something that should make it ultra hard to upgrade. I think we should do this sooner rather than later, as part of @helenwh 's change to generally make variable value filtering more powerful.